gui-backend/src/main/java/com/rln/gui/backend/implementation/methods/AutoapproveApiImpl.java

```diff
@@ -80,6 +84,25 @@ public List<LedgerAddressDTO> get() {
     return result;
   }
 
+  public List<WalletAddressDTO> getForWallet() {
```

Comment:
The method name suggests an argument but there is none.
Also, write a quick description above the method definition.

Reply:
What method name would you suggest?
I think I broke the "pattern" because the original idea was to implement the logic/core part of the API calls in different Impl classes and just call them from here, so I should have called this `get3`, what do you think about that (it is weird but that's the operationId somehow)?

Reply:
I see. I would still just name it `getWalletAddresses`

Reply:
I renamed it